datajunction-server/datajunction_server/api/tags.py

```diff
@@ -144,5 +163,8 @@ def list_nodes_for_a_tag(
             http_status_code=404,
         )
     if not node_type:
-        return sorted([node.name for node in tag.nodes])
-    return sorted([node.name for node in tag.nodes if node.type == node_type])
+        return sorted([node.current for node in tag.nodes], key=lambda x: x.name)
+    return sorted(
```

Comment:
Do you think it's better to leave sorting to the frontend (UI, clients, etc)? Tags may not grow quickly enough for there to be a large number of them but when that happens, I'm wondering about centralizing the sort to the backend server vs. distributing it among the clients. Either most likely will work for a looong time so maybe it's not worth worrying about right now.

Reply:
I agree on this. Seems nice but unnecessary.

Reply:
Good point, might be easier to just stick to client-side sorting. Although I was thinking that if we have a defined order, it might be simpler for the server to always maintain that order (i.e., column order on a node). In the case of tags though, alphabetical order is somewhat of an arbitrary choice.